cmd/argocd/commands/root.go

```diff
@@ -69,5 +69,6 @@ func NewCommand() *cobra.Command {
 	command.PersistentFlags().StringSliceVarP(&clientOpts.Headers, "header", "H", []string{}, "Sets additional header to all requests made by Argo CD CLI. (Can be repeated multiple times to add multiple headers, also supports comma separated headers)")
 	command.PersistentFlags().BoolVar(&clientOpts.PortForward, "port-forward", config.GetBoolFlag("port-forward"), "Connect to a random argocd-server port using port forwarding")
 	command.PersistentFlags().StringVar(&clientOpts.PortForwardNamespace, "port-forward-namespace", config.GetFlag("port-forward-namespace", ""), "Namespace name which should be used for port forwarding")
+	command.PersistentFlags().IntVar(&clientOpts.RetryMax, "retry-max", 0, "Maximum number of retries")
```

Comment:
I don't think `retry-max` is a good name for the option. The name should imply _what_ is actually retried, although I cannot think of a _proper_ name right now. Maybe `http-retries` or `api-retries` would fit better? Also, the description should be adapted to be a little more verbose.

Reply:
Renamed to `http-retry-max` and adapted the description to be more verbose.